src/components/Card/LandingPageCard.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import propTypes from 'prop-types';
+const LandingPageCard = ({ image, title }) => {
+  return (
+    <>
+      <div className=" max-h-lg max-w-sm sm:w-1/2 lg:w-1/4  overflow-hidden shadow-lg transition duration-300 ease-in-out transform hover:-translate-y-1 hover:scale-105 rounded">
+        <img className="w-full" src={image} alt="Train to Maintain landing" />
```

Comment:
can we make the image width and height fixed? cause whenever you change the image the card will change according to and we don't really want to see that, also make sure you make it `fit` right like make the fit `cover`